redaxo/src/core/pages/profile.php

```diff
@@ -39,10 +39,9 @@
 $locales = rex_i18n::getLocales();
 asort($locales);
 foreach ($locales as $locale) {
-    rex_i18n::setLocale($locale, false); // Locale nicht neu setzen
-    $sel_be_sprache->addOption(rex_i18n::msg('lang'), $locale);
+    $sel_be_sprache->addOption(rex_i18n::msgInLocale('lang', $locale), $locale);
 }
-rex_i18n::setLocale($userperm_be_sprache, false);
```

Comment:
das `false`  hier unten war meiner meinung nach ein fehler.

Reply:
Wird die Zeile überhaupt noch benötigt?
Es ging doch hier nur darum, die Sprache wieder zurück zu stellen, oder?

Reply:
stimmt, werf ich raus.

Reply:
nein, hier wird die sprache für die aktuelle seite eingestellt

Reply:
d.h. ohne diese zeile braucht es einen weiteren reload um die sprachänderung zu sehen

Reply:
hmm, sind dann nicht schon manche parts vorher in der anderen sprache übersetzt worden?

Reply:
stimmt.. einzelne texte sind falsch. dann lieber nen reload immer notwendig, oder?

Reply:
done